rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/report/SW360ReportController.java

```diff
@@ -151,43 +158,58 @@ private void getComponentsReports(boolean withLinkedReleases, boolean mailReques
 
     private void getLicensesReports(HttpServletResponse response, User sw360User, String module) throws TException {
         try {
-            downloadExcelReport(false, response, sw360User, module, null);
+            downloadExcelReport(false, response, sw360User, module, null, defaultByteBufferVal);
         } catch (Exception e) {
             throw new TException(e.getMessage());
         }
     }
 
-    private void downloadExcelReport(boolean withLinkedReleases, HttpServletResponse response, User user, String module, String projectId)
-            throws TException {
+    private void getLicenseResourceBundleReports(String projectId, HttpServletRequest request,
+            HttpServletResponse response, User sw360User, String module) throws TException {
         try {
-            ByteBuffer buffer = null;
+            ByteBuffer buffer = sw360ReportService.downloadSourceCodeBundle(projectId, request, sw360User);
+            downloadExcelReport(false, response, sw360User, module, null, buffer);
```

Comment:
The `projectId` is passed here as `null` to `downloadExcelReport()` which throws an exception while calling `sw360ReportService.getSourceCodeBundleName()` as it cannot get the project from DB.

Tested on a project with no attachment usage and following request
`GET http://{{ host }}/resource/api/reports?module=licenseResourceBundle&isAllAttachmentSelected=true&projectId=<proj_id>`

Reply:
Resolved